paasta_itests/steps/setup_chronos_job_steps.py

```diff
@@ -30,12 +30,21 @@ def job_exists(context, service, instance):
     assert len(matching_jobs) == 1
 
 
-@when(u'we run setup_chronos_job for service_instance "{service_instance}"')
-def run_setup_chronos_job(context, service_instance):
+@when(u'we run setup_chronos_job for service_instance "{service_instance}", expecting exit code {expected_exit_code}')
+def run_setup_chronos_job(context, service_instance, expected_exit_code):
     cmd = "../paasta_tools/setup_chronos_job.py %s -d %s" % (service_instance, context.soa_dir)
     exit_code, output = _run(cmd)
     context.exit_code, context.output = exit_code, output
 
+    try:
+        assert context.exit_code == int(expected_exit_code), \
```

Comment:
you can tell behave the type rather than having to cast. something like {expected_exit_code:d}